pages/community/precice-workshop-2024.md

```diff
@@ -31,20 +31,22 @@ The registration for the workshop will open in the next few weeks. You will be a
 
 #### Early-bird registration
 
-| Ticket type   | Members of academia | Members of industry  |
-|--------------|-----------|------------|
-| **full registration** | 300 Euro    | 500  Euro     |
-| **main part only**      | 200 Euro  | 300 Euro       |
+| Ticket type | Members of academia | Members of industry  |
+|---|---|---|
+| **workshop + course** | 300 Euro    | 500  Euro     |
+| **workshop**      | 200 Euro  | 300 Euro       |
 
 #### Standard registration
 
-| Ticket type   | Members of academia | Members of industry  |
-|--------------|-----------|------------|
-| **full registration** | 450 Euro    | 750  Euro     |
-| **main part only**      | 300 Euro  | 500 Euro       |
+| Ticket type | Members of academia | Members of industry  |
+|---|---|---|
+| **workshop + course** | 450 Euro    | 750  Euro     |
+| **workshop**      | 300 Euro  | 500 Euro       |
 
 If you are a student and struggle to find funding, you might be eligible for our student travel awards.
 
+Holders of a [preCICE suport license](https://precice.org/community-support-precice.html#2-support-license) are excempt of the registration fee and can register to the workshop by simply confirming assistance to preCICE2024@simtech.uni-stuttgart.de.
```

Comment:
Not sure if "assistance" is the right word here.
Still important to emphasize when they need to do that (late registration deadline, probably).